azkaban-common/src/main/java/azkaban/executor/container/ContainerCleanupManager.java

```diff
@@ -363,7 +364,8 @@ void cleanUpYarnApplicationsInCluster(Set<Integer> toBeCleanedContainers,
           aliveApplications.stream().map(app -> app.getApplicationId().toString())
               .collect(Collectors.joining(",")));
     } catch (Exception e) {
-      logger.error("fail to get yarn applications by execution IDs", e);
+      logger.error("fail to get yarn applications by execution IDs from cluster "
```

Comment:
Do we need to emit any similar metrics here?
If thread failed due to some random reason here, markYarnApplicationKillFail will not be called.

Reply:
good point, adding one now